tools/osbuild-mpp

```diff
@@ -686,7 +689,8 @@ class ManifestFile:
                 format_string = node["mpp-format-int"]
 
             # pylint: disable=eval-used  # yolo this is fine!
-            res = eval(f'f\'\'\'{format_string}\'\'\'', local_vars)
+            # Note, we copy local_vars here to avoid eval modifying it
+            res = eval(f'f\'\'\'{format_string}\'\'\'', dict(local_vars))
```

Comment:
It would be nice to have this in a separate commit (in the same PR), as it is not necessarily related to the feature and is useful even without it.

Reply:
can do

Reply:
Done